WrightTools/data/_data.py

```diff
@@ -917,8 +919,10 @@ def create_variable(
                 shape = self.shape
             if dtype is None:
                 dtype = np.dtype(np.float64)
-            if dtype.kind in "fcmM":
+            if dtype.kind in "fmM":
```

Comment:
Actually, now that I think about this, does dtype "m", "M" support `np.nan`?
```
x = np.zeros((3), dtype=np.timedelta64)
x[0] = np.nan
```
This throws an error...fill values for time should be NaT?

Reply:
Since I do not know what kind of data arrays we use require "m" kind for our purposes, I took that kind out